maps_dashboards/public/model/documentLayerFunctions.ts

```diff
@@ -58,39 +58,46 @@ const getGeoFieldName = (layerConfig: DocumentLayerSpecification) => {
   return layerConfig?.source?.geoFieldName;
 };
 
+const buildGeometry = (fieldType: string, location: any) => {
+  if (fieldType === 'geo_point') {
+    return {
+      type: 'Point',
+      coordinates: [location.lon, location.lat],
```

Comment:
Should the first value in array be "lat" and second one "lon"?

Reply:
The order is longitude and latitude. It's the definition from GeoJSON.
>  A position is an array of numbers.  There MUST be two or more
   elements.  The first two elements are longitude and latitude, or
   easting and northing, precisely in that order and using decimal
   numbers.  Altitude or elevation MAY be included as an optional third
   element. 

Ref: https://www.rfc-editor.org/rfc/rfc7946#section-3.1.1